### PR TITLE
Hotkey mode handling is now instantaneous

### PR DESCRIPTION
**Pull request by Markolie**
**What does this PR do:**
This pull request reworks https://github.com/ParadiseSS13/Paradise/pull/121. Hotkey mode toggling has been moved back to the client side, which means that toggling it is now instantaneous. There is also no longer any lag when switching between modes.

Support for AZERTY has been maintained: the "Set Hotkey Mode" option is still present in the preferences menu.

Fixes https://github.com/ParadiseSS13/Paradise/issues/5841.

**Changelog:**
:cl: Markolie
fix: Changing your hotkey mode now happens instantaneously and no longer lags.
/:cl:

### CODE REVIEW

**Comment by Markolie** [comment]
Spelling is hard

**Comment by Spacemanspark** [comment]
...Holy heck.

We don't deserve Mark.

**Comment by Markolie** [comment]
I may have thought of a solution to let the frogs keep their AZERTY, will look into that in an hour or so.

**Comment by Markolie** [comment]
Support for AZERTY has been restored!

File: code/modules/mob/camera/camera.dm [hunk 1]
```diff
@@ -17,4 +17,3 @@
 
 /mob/camera/Login()
 	..()
```
Comment (Fox-McCloud) [inline code/modules/mob/camera/camera.dm]: no point to this now.

**Pull request merged**